WhitmanCSCapstone/ge-scheduling
# Output results to sheet

Building off of #42. Add back the feature of outputting to the output spreadsheet.

## Testing
- Push this branch to your Apps Script dev project.
- Run `main`.
- See that the output sheet is populated with names. Students aren't yet being assigned preferences.
